public/app.js

```diff
@@ -15,35 +15,41 @@ function dietPlan() {
 		factor = .2;
 	}
 	responsiveSize = width * factor;
-	cat.style.width = `${responsiveSize}px`;
-	cat.style.height = `${responsiveSize}px`;
+	for(let i = 0, len = cats.length; i < len; i++) {
```

Comment:
Move the len declaration out of the loop parameters to make it a bit cleaner. Just pop it up one line

Reply:
Please verify now

Reply:
Looks good!